### ISSUE TITLE
Support for authorizing VSTS in Azure Key Vault task

### ISSUE THREAD

**Issue by Mardoxx** [microsoft/azure-pipelines-tasks#8073]
Would be nice to have ACL for selected keyvault auto provisioned.

Have to do this workaround, adding SP manually failed every time because I don't know what I'm doing and documentation is sparse at best!!
Please see https://stackoverflow.com/questions/50968613/cannot-acces-keyvault-secrets-through-service-endpoint-in-a-vsts-release/51931131#51931131

**Comment by lovakumar** [comment]
@Mardoxx provisioning of ACL for selected key vault needs user authentication hence it needs to go through user interaction flow only.

**Comment by Mardoxx** [comment]
Why have you closed this? :(

Currently there is no way to auth vsts with the kv within the task

**Comment by lovakumar** [comment]
@Mardoxx Link that you shared and issue description sounded like you are looking for auto provisioning of ACL in key vault variable group.

If you are looking for similar thing in Azure Key Vault task, we currently generate a scrip as part of the task logs that you can run to provision ACL.

Inline support in task (similar variable group) is not in our backlog yet. Can you please try using the script generated by task. If that is not helping then we can add inline support for auto provisioning ACL in our backlog.

**Comment by Mardoxx** [comment]
Sorry should have been more clear!!

Digging through logs only after pipeline has failed is not great imo.
Inline Auth in task would be perfect!

**Comment by lovakumar** [comment]
Ok got it. @N-Usha @RoopeshNair can you please add this feature to our backlog.

**Comment by N-Usha** [comment]
Added to the backlog and will update as we make progress in addressing it. Thanks!

**Comment by dazinator** [comment]
No news on this?

**Comment by dazinator** [comment]
- I'm using the task from Azure Devops, and hit #10274

**Comment by leantk** [comment]
This issue is stale because it has been open for a year with no activity. Remove the stale label or comment on the issue otherwise this will be closed in 5 days

**Comment by dazinator** [comment]
#dontclose
